src/humancellatlas/data/metadata/helpers/dss.py

```diff
@@ -36,9 +52,15 @@ def download_bundle_metadata(client: DSSClient,
              in the bundle (data and metadata) and a dictionary mapping the file name of each metadata file in the
              bundle to the JSON contents of that file.
     """
+    if directurls or presignedurls:
```

Comment:
We should also enforce `not ( directlurls and presignedurls)`.

Reply:
I didn't add that in the first place for `not (directlurls and presignedurls)` are guaranteed by the dss client, which will throw an error:

```
requests.codes.bad_request, "only_one_urltype", "only enable one of `directurls` or `presignedurls`"
``` 

and it looked redundant to me. I can definitely add it to here now for the completeness of a standalone lib.

Reply:
What do you think would be the best way to enforce `not (directlurls and presignedurls)` here? How about raising an exception when (directlurls and presignedurls)?

Reply:
> How about raising an exception when (directlurls and presignedurls)

Maybe. I think we should let DSS reject it as you proposed and not enforce the restriction here. What if DSS decides to allow both parameters to be true at some point?